protocols/kad/src/protocol.rs

```diff
@@ -179,6 +177,47 @@ impl UpgradeInfo for KademliaProtocolConfig {
     }
 }
 
+/// Codec for Kademlia inbound and outbound message framing.
+pub struct KadCodec<A, B> {
```

Comment:
```suggestion
pub struct Codec<A, B> {
```

We are within the scope of `libp2p-kad` so I'd say this is redundant. Related: https://github.com/libp2p/rust-libp2p/issues/4485.

Reply:
got it. fixed now